src/GDGUkraine/rest_controller.py

```diff
@@ -372,77 +373,65 @@ def export_participants(self, id):
-        exporter = TableExporter(
-            data=participations,
-            data_getters=[
-                (lambda x: x.EventParticipant.id),
-                (lambda x: x.EventParticipant.register_date),
-                (lambda x: id),
-                (lambda x: event.title),
-                (lambda x: x.User.full_name),
-                (lambda x: x.User.gender),
-                (lambda x: x.User.nickname or ''),
-                (lambda x: x.User.email),
-                (lambda x: x.User.phone or ''),
-                (lambda x: x.User.gplus),
-                (lambda x: x.User.hometown or ''),
-                (lambda x: x.User.company or ''),
-                (lambda x: x.User.position or ''),
-                (lambda x: x.User.www or ''),
-                (lambda x: x.User.experience_level or ''),
-                (lambda x: x.User.experience_desc or ''),
-                (lambda x: x.User.interests or ''),
-                (lambda x: x.User.events_visited or ''),
-                (lambda x: x.User.english_knowledge or ''),
-                (lambda x: x.User.t_shirt_size or ''),
-                (lambda x: x.User.additional_info or ''),
-                (lambda x: (
-                    json.dumps(x.EventParticipant.fields)
-                    if x.EventParticipant.fields
-                    else ''
-                )),
-                (lambda x: x.EventParticipant.confirmed),
-            ],
-            headers=[
-                'Registration id',
-                'Registration date',
-                'Event id',
-                'Event title',
-                'Name',
-                'Gender',
-                'Nickname',
-                'Email',
-                'Phone',
-                'Google +',
-                'City',
-                'Company',
-                'Position',
-                'Website',
-                'Experience level',
-                'Experience description',
-                'Interests',
-                'Events visited',
-                'English knowledge',
-                'T-Shirt size',
-                'Additional info',
-                'Extra fields',
-                'Confirmed',
-            ],
-        )
+
+        # Set appropriate headers
+        filename = re.compile(r'[^\w-]').sub('', event.title.replace(' ', '_'))
+
         cherrypy.response.headers['Content-Type'] = (
             'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
         )
-        filename = re.compile(r'[^\w-]').sub('', event.title.replace(' ', '_'))
         cherrypy.response.headers['Content-Disposition'] = (
             'attachment; filename={}-{}-{}-participants.xlsx'.format(
                 event.id, filename, event.date,
             )
         )
-        return file_generator(exporter.get_xlsx_content())
+
+        # Retrieve participation data
+        part_data = api.find_participants_by_event(orm_session, event)
```

Comment:
I spent about 3mins trying to understand how do you split participations into parts until I realized that part is a shorthand for participation. Rename variable.